.editorconfig

```diff
@@ -97,3 +97,9 @@ csharp_style_pattern_matching_over_as_with_null_check = true:suggestion
 csharp_style_inlined_variable_declaration = true:none
 csharp_style_throw_expression = true:none
 csharp_style_conditional_delegate_call = true:suggestion
+
+# Prefer generally simpler code
+dotnet_diagnostic.IDE0005.severity = warning      # Remove unnecessary import
```

Comment:
According to EditorConfig.org, placement of a "comment" at the end of a line has unspecified behavior. Comments should always precede the configuration in this file.